webapp/channels/src/components/admin_console/admin_console.test.tsx

```diff
@@ -87,8 +88,8 @@ describe('components/AdminConsole', () => {
             consoleAccess: {read: {}, write: {}},
             team: {name: 'development'} as Team,
         };
-        const wrapper = shallow(
-            <AdminConsole {...props}/>,
+        const wrapper = renderWithIntl(
+            <BrowserRouter><AdminConsole {...props}/></BrowserRouter>,
         );
         expect(wrapper).toMatchSnapshot();
```

Comment:
To have a meaningful snapshot here, you should mock the Redirect, ex.:
```javascript
jest.mock('react-router-dom', () => {
    const originalModule = jest.requireActual('react-router-dom');

    return {
        ...originalModule,
        Redirect: jest.fn(({to}) => `Redirect to=${to}`),
    };
});
```

Then this could be:
```javascript
const wrapper = renderWithIntl(
    <div id='root'><BrowserRouter><AdminConsole {...props}/></BrowserRouter></div>,
);
expect(wrapper.container).toMatchSnapshot();
```

Would be good to wrap the component with the expected base element so you'll see some CSS applied to root element.